modules/Microsoft.DBforPostgreSQL/flexibleServers/.test/min/deploy.test.bicep

```
@@ -0,0 +1,45 @@
+targetScope = 'subscription'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'ms.dbforpostgresql.flexibleservers-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'dpsqlfsmin'
```

Is this aligned with other serviceshort parameter values or should it be
```suggestion
param serviceShort string = 'dbfpsqlfsmin'
```
instead? Or maybe `dfpsmin`?